api/src/main/java/org/openmrs/ConditionClinicalStatus.java

```diff
@@ -0,0 +1,41 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+
+package org.openmrs;
+
+/**
+ * The ConditionClinicalStatus is what we've historically called "condition status" and what FHIR refers to as
+ * ConditionClinicalStatus . This is a clinical condition that has risen to a level of concern.
+ * 
+ * <li>{@link #ACTIVE}</li>
+ * <li>{@link #INACTIVE}</li>
+ * <li>{@link #HISTORY_OF}</li>
+ *
+ * */
+
+public enum ConditionClinicalStatus {
+	/**
+	 * This is where the subject is currently experiencing the symptoms of the condition or there is evidence of the 
```

Comment:
Can we also change from subject to patient?

Reply:
Done